### PR TITLE
Sync with fork and re-apply buildx

### PR DESCRIPTION
**Pull request by antogon**
We can't merge this -- we should replace `master` with this branch once we've finished testing the latest images I just published for our runners in GCR.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ charts
 *.md
 *.txt
 *.sh
+test/e2e/.docker-build
```
Comment (matrinox) [inline .dockerignore:14]: Not sure if we want to get aggressive with removing files but here's a few more:
```
*.stories.*
js-packages/mobile-app
docs/
.storybook/
.github/
extension/ # I think it's not being used by our Dockerfile
```

**Comment by jordi-chacon** [comment]
@antogon Is this PR still needing some work on our side?

**Comment by antogon** [comment]
> @antogon Is this PR still needing some work on our side?

We need to replace master with this branch, not merge this PR.

**Comment by jordi-chacon** [comment]
> We need to replace master with this branch, not merge this PR.

@antogon @matrinox What's preventing us from doing this? Do we still want this PR?

**Pull request closed without merging**